src/sbin/prefixes.php

```diff
@@ -60,7 +60,8 @@
 // echo "add routes\n";
 if (count($routes) > 0) {
     foreach ($routes as $address => $prefix) {
-        echo "/sbin/route change -inet6 {$prefix} {$address}\n";
```

Comment:
hmmm, I think this is it, "route change" was a custom patch a long time ago in a project far far away